tasks/deprecated-image.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: deprecated-image
+spec:
+  stepTemplate:
+    volumeMounts:
+      - name: data
+        mountPath: /data
+
+  params:
+    - name: source-image
+      description: "Image name to test"
+    - name: policy-dir
+      description: "Path to the directory containing conftest policies"
+      default: "/project/image/policy/deprecated-image.rego"
+
+  results:
+    - name: pyxis-http-code
+      description: HTTP code returned by Pyxis API endpoint
+
+  steps:
+    # Download Pyxis metadata about the image
+    #
+    # project
+    # └── policy
+    # data
+    # └── image_data.json
+    - name: query-pyxis
+      image: registry.access.redhat.com/ubi8/ubi
+      script: |
+        #!/usr/bin/env bash
+        echo "Querying Pyxis for $(inputs.params.source-image)..."
+        http_code=$(curl -s -o /data/image_data.json -w '%{http_code}' "https://catalog.redhat.com/api/containers/v1/repositories/registry/registry.access.redhat.com/repository/$(inputs.params.source-image)")
```

Comment:
Maybe have the `registry.access.redhat.com` as a parameter to allow better flexibility in future (e.g. the ISV images have a diferent value here).

Reply:
+1, we may have different registry

Reply:
Done!